app/controllers/concerns/hyku_addons/users_controller_behavior.rb

```diff
@@ -12,7 +12,15 @@ def index
 
     def show
       @user = User.find_by(email: params[:email], display_profile: true)
-      render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?
+      render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } and return if @user.blank?
```

Comment:
This is pretty complicated: 

```
if @user.blank?
    return render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } 
end
````

Reply:
Your suggestion fails RuboCop::Cop::Style::IfUnlessModifier

Reply:
I dont know what that means. What is the suggested alternative?

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/IfUnlessModifier

Basically exactly as I have it

Reply:
My issues wasn't with the end of line `if/unless`, it was the `and return`. 

Why not `return render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?` ?

Reply:
Ill give it a go

Reply:
'do_soemthing and return` is different from `return do_something`.

Reply:
To clarify the situation: if you don't add a return statement here do you get a double render error? If that's the case instead of using `render {} and return` you should try to construct this so you don't double render.  I presume this is happening because you specifically want to render an error message and return a 403.  In that case you need to correctly override the other bit of code that calls render so that it is not called if `@user.blank?`.

Reply:
I don't think there was actually a double render error as I thought originally 🙈🙈. I think i remembered using the `and return` in the past to stop code running and just applied it here. 

The actual reason to return is so that we don't call `@user.email ` if there is no user